lib/argot/traject_writer.rb

```diff
@@ -6,7 +6,7 @@ module Argot
   # top-level attributes of the current record to scalars, which is
   # the form expected by Argot.
   class TrajectJSONWriter < Traject::JsonWriter
-    @array_fields = %w[holdings url items].to_set
+    @array_fields = %w[holdings url items imprint_main imprint_multiple].to_set
```

Comment:
I'm pretty unfamiliar with this code, so pls forgive obvious questions... 

Is `@array_fields` here the list of Argot fields that take the structured data approach? The ones that are an array of hashes or whatever? 

This isn't any Argot field where the output is itself an array, like `publisher`, etc., correct?

Reply:
@kspurgin Yes, that holds an array of fields to treat as structured data (like url, not like publisher).

Reply:
Any fields not in that array get flattened if it contains any structure.